src/Symfony/Component/Filesystem/Tests/PathTest.php

```diff
@@ -589,7 +605,7 @@ public function testMakeAbsoluteDoesNotFailIfDifferentRoot(string $basePath, str
 
     public function provideMakeRelativeTests(): \Generator
     {
-        foreach ($this->providePathTests() as $set) {
+        foreach (static::PATH_TESTS as $set) {
```

Comment:
To use a constant as "provider" is quite uncommon. I would go with a static method instead, but no strong opinion

Reply:
I don't have a strong opinion on this one either, but I prefer the static method way 👍 Updated!